### PR TITLE
Help Command: /helpsavebuilds

### PR DESCRIPTION
**Pull request by ghost**
Helping players how to save their builds

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_helpsavebuilds [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.BOTH, blockHostConsole = true)
```
Comment (rsajeey) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_helpsavebuilds]: I don't think "blockHostConsole" is needed.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_helpsavebuilds [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+@CommandParameters(description = "Tell players how to save their builds", usage = "/<command>")
+public class Command_helpsavebuilds extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        sender.sendMessage(ChatColor.GREEN + "1, Type: //wand");
```
Comment (rsajeey) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_helpsavebuilds]: Use "playerMsg" instead of "sender.sendMessage".

Comment (Adamx120) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_helpsavebuilds]: Done

**Comment by JeromSar** [comment]
No associated issue? Yet again, I have to put this issue on hold until there is.
Additionally, this PR does not compile.

I keep telling you, but I'll repeat myself. Before you make any more PRs, or open any more issues. please review the [contributing guidelines](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing).

**Comment by OnlineRepz** [comment]
You are missing .java extension.

**Comment by Adamx120** [comment]
@JeromSar https://github.com/TotalFreedom/TotalFreedomMod/issues/995?

**Comment by JeromSar** [comment]
This PR does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and thus is closed.

**Pull request closed without merging**